stream.go

```diff
@@ -145,7 +144,7 @@ func (cc *ClientConn) NewStream(ctx context.Context, desc *StreamDesc, method st
 
 // NewClientStream is a wrapper for ClientConn.NewStream.
 //
-// DEPRECATED: Use ClientConn.NewStream instead.
+// Deprecated: Use ClientConn.NewStream instead.
```

Comment:
Let's not change this just yet...generated code still uses this (so vet fails).

Reply:
Done. Ditto for ServerStream and Stream?